static/bidder-info/conversant.yaml

```diff
@@ -9,10 +9,12 @@ capabilities:
     mediaTypes:
       - banner
       - video
+      - audio
   site:
     mediaTypes:
       - banner
       - video
+      - audio
```

Comment:
Please add exemplary jsons tests for the app audio and site audio.

Reply:
ok, done.  I should have done that before